test/functional/signrawtransactions.py

```diff
@@ -138,6 +139,35 @@ def run_test(self):
         self.successful_signing_test()
         self.script_verification_error_test()
 
+        # Now test signing transaction to P2SH-P2WSH addresses without wallet
+        # Create a new P2SH-P2WSH 1-of-1 multisig address:
+        embedded_address = self.nodes[1].getnewaddress()
+        embedded_pubkey = self.nodes[1].validateaddress(embedded_address)["pubkey"]
+        embedded_privkey = self.nodes[1].dumpprivkey(embedded_address)
+        multisig_address = self.nodes[1].addmultisigaddress(1, [embedded_pubkey])
+        p2sh_p2wsh_address = self.nodes[1].addwitnessaddress(multisig_address)
+        witness_script = CScript([OP_1, hex_str_to_bytes(embedded_pubkey), OP_1, OP_CHECKMULTISIG])
+
+        # send transaction to P2SH-P2WSH 1-of-1 multisig address
+        self.nodes[0].generate(101)
+        tx_to_witness = self.nodes[0].createrawtransaction([self.nodes[0].listunspent()[0]], {p2sh_p2wsh_address: Decimal("49.999")})
+        signed_tx_to_witness = self.nodes[0].signrawtransaction(tx_to_witness)
+        assert ("errors" not in signed_tx_to_witness or len(["errors"]) == 0)
+        self.nodes[0].sendrawtransaction(signed_tx_to_witness["hex"])
+        self.nodes[0].generate(1)
+        self.sync_all()
+
+        # Find the UTXO for the transaction node[1] should have received, check witnessScript matches
+        unspent_output = self.nodes[1].listunspent(0, 999999, [p2sh_p2wsh_address])[0]
+        assert_equal(unspent_output["redeemScript"][1], bytes_to_hex_str(witness_script))
```

Comment:
nit: check if the redeemScript[0] is the p2sh redeem.

Reply:
Done